### PR TITLE
Feature/idva5 1464 update where you live

### PR DESCRIPTION
**Pull request by hemerem-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/features/sole-trader/where-do-you-live/where-do-you-live.njk [hunk 1]
```diff
@@ -52,14 +52,28 @@
...
-      id:"save-continue-button"
-    }) }}
+
+{% if reqType === "updateAcsp" %}
+          <div class="govuk-button-group">
+             {{ govukButton({
+                text: i18n.Continue,
+                id:"continue-button-id"
+             })}}
+             <a class="govuk-link id="previous-page-id" href={{previousPage}}>{{i18n.Cancel}}</a>
```
Comment (cbroadley-ch) [inline src/views/features/sole-trader/where-do-you-live/where-do-you-live.njk:62]: Looks like you are missing a closing " on the class of this anchor tag.
Would need to update it like so:

<a class="govuk-link**"** id="previous-page-id" href={{previousPage}}>{{i18n.Cancel}}</a>

File: src/controllers/features/update-acsp/whereDoYouLiveController.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { saveDataInSession } from "../../../common/__utils/sessionHelper";
+import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
+import { WhereDoYouLivBodyService } from "../../../services/where-do-you-live/whereDoYouLive";
+import { USER_DATA } from "../../../common/__utils/constants";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const acspData: AcspData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
```
Comment (sampankumar) [inline src/controllers/features/update-acsp/whereDoYouLiveController.ts:18]: Why do we need acspData to have some value?  I believe we can just do this.
const acspData: AcspData = session?.getExtraData(USER_DATA)!;

And handle the null of acspData separately

Comment (sampankumar) [reply src/controllers/features/update-acsp/whereDoYouLiveController.ts:18]: Can we please add a comment in the code and also raise a JIRA ticket as as tech debt to do this later.

**Review by sardhani-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Pull request merged**